.github/workflows/ci.yaml

```diff
@@ -0,0 +1,131 @@
+# Code generated by dhall-to-yaml.  DO NOT EDIT.
+jobs:
+  checkPipeline:
+    name: render-CI-pipeline
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: 'actions/checkout@v2'
+      - name: Install asdf
+        uses: 'asdf-vm/actions/setup@v1.0.1'
+      - name: Install asdf plugins
+        run: './scripts/asdf-add-plugins.sh'
+      - name: Install asdf tool versions and packages
+        run: './ci/asdf-install.sh'
+      - name: Check that the CI pipeline definition is up-to-date with the dhall configuration
+        run: ci/check-rendered-pipeline-up-to-date.sh
+  dhallCheck:
+    name: dhall-check
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: 'actions/checkout@v2'
+      - name: Install asdf
+        uses: 'asdf-vm/actions/setup@v1.0.1'
+      - name: Install asdf plugins
+        run: './scripts/asdf-add-plugins.sh'
+      - name: Install asdf tool versions and packages
+        run: './ci/asdf-install.sh'
+      - name: Check that all dhall files typecheck
+        run: just check-dhall
+  dhallFormat:
+    name: dhall-format
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: 'actions/checkout@v2'
+      - name: Install asdf
+        uses: 'asdf-vm/actions/setup@v1.0.1'
+      - name: Install asdf plugins
+        run: './scripts/asdf-add-plugins.sh'
+      - name: Install asdf tool versions and packages
+        run: './ci/asdf-install.sh'
+      - env:
+          CHECK: 'true'
+        name: Check that dhall files are formatted
+        run: just format-dhall
+  dhallLint:
+    name: dhall-lint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: 'actions/checkout@v2'
+      - name: Install asdf
+        uses: 'asdf-vm/actions/setup@v1.0.1'
+      - name: Install asdf plugins
+        run: './scripts/asdf-add-plugins.sh'
+      - name: Install asdf tool versions and packages
+        run: './ci/asdf-install.sh'
+      - env:
+          CHECK: 'true'
+        name: Check that dhall files are linted properly
+        run: just lint-dhall
+  golangciLint:
+    name: golangci-lint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: 'actions/checkout@v2'
+      - name: Install asdf
+        uses: 'asdf-vm/actions/setup@v1.0.1'
+      - name: Install asdf plugins
+        run: './scripts/asdf-add-plugins.sh'
+      - name: Install asdf tool versions and packages
+        run: './ci/asdf-install.sh'
+      - name: 'golangci-lint (.)'
+        uses: 'golangci/golangci-lint-action@v2.5.2'
+        with:
+          version: v1.39.0
+          working-directory: '.'
+  prettier:
```

Comment:
same question here

Reply:
Prettier is used to autoformat all javascript, json, yaml, and markdown files in this repo.